docs/appendices/release-notes/unreleased.rst

```diff
@@ -43,6 +43,10 @@ Unreleased Changes
 Breaking Changes
 ================
 
+- Changed ``pg_attribute.spcacl``, ``pg_class.relacl`` and
+  ``pg_namespace.nspacl`` columns type from ``OBJECT[]`` to ``STRING[]`` to be
+  in sync with other similar columns, e.g.: ``pg_database.datacl``.
```

Comment:
Wouldn't this also improve PG compatibility?

Reply:
I'm not sure that this has an actual effect, since we always return null anyway, it's more to be consistent in all such columns.